src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1435,3 +1435,80 @@ function CategoricalArrays.categorical(df::AbstractDataFrame,
     end
     DataFrame(newcols, _names(df), copycols=false)
 end
+
+"""
+    flatten(df::AbstractDataFrame, cols::Union{Integer, Symbol})
+
+When column `cols` of `df` has iterable elements that define `length`, for example a `Vector` 
```

Comment:
This sentence has no verb. Better start with "Return" from the second sentence and include this part in it?